lib/main_common.pm

```diff
@@ -2903,9 +2841,8 @@ sub load_transactional_role_tests {
 
 sub load_common_opensuse_sle_tests {
     load_autoyast_clone_tests if get_var("CLONE_SYSTEM");
-    load_publiccloud_tests if get_var('PUBLIC_CLOUD');
     loadtest "terraform/create_image" if get_var('TERRAFORM');
-    load_create_hdd_tests if get_var("STORE_HDD_1") || get_var("PUBLISH_HDD_1");
+    load_create_hdd_tests if (get_var("STORE_HDD_1") || get_var("PUBLISH_HDD_1")) && !get_var('PUBLIC_CLOUD');
```

Comment:
Can you explain why we need this additional condition ?

Reply:
no tests which has `PUBLIC_CLOUD` has `PUBLISH_HDD_1` or `STORE_HDD_1`

Reply:
i believe the reason was that it was messing with the scheduling. need to check. i do not really remember.

Reply:
So the job which has the var `PUBLISH_HDD_1` and breaks the scheduling is `publiccloud_download_testrepos`.

Reply:
ok I see , so let's keep than